spec/system/admin/order_spec.rb

```diff
@@ -992,6 +994,8 @@ def new_order_with_distribution(distributor, order_cycle)
             expect(page).to have_text 'SHIPPED'
           end
         end
+        
+        it_behaves_like "ship order from dropdown", "Customer Details", "Order Details", "Payments", "Adjustments", "Invoices", "Return Authorizations"
```

Comment:
I think that we should change our linter and instead of only looking at the diff, we should run Rubocop on the whole codebase as we would as a developer. It's easier to reproduce and catches violations when rules change in master while a branch is developing. Or when we change rules in a branch then it tests the whole code.

Reply:
> should change our linter and instead of only looking at the diff, we should run Rubocop on the whole codebase as we would as a developer

Hum, I'm not sure I fully understand, as the file above was changed and as such, it should appear on the diff... In any case, I've opened an issue [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/12389), so this suggestion does not get lost.